tests/data/legacy/implode_hamlet_256.bin

```diff
@@ -0,0 +1,2 @@
+
#67h���<D��������P!T�oՖK�Ѻt˺[V�Y�E��}��[� ��m˶,]� ��y5-[�i�7ȩhî-;nٰr���Mmڹ ˼7�7�V.H�h˂�;����6�[�e�����·��^�m�tM&Y�
c��t?c�����
��o8ٚ���>�@k6��|�t����~���A�l����_�
```

Comment:
Git still thinks this is a text file.

Reply:
should be fixed.

Reply:
According to .gitattributes that doesn't make much sense - added whole legacy directory as binary.